grc/blocks/cpp_hb_nogui.workflow.yml

```diff
@@ -0,0 +1,25 @@
+id: cpp_hb_nogui_workflow
+description: "c++ hier block no gui workflow"
```

Comment:
"C++" should probably be capitalized here, same for "Python" in the other workflows; not sure about gui/GUI, I'd tend towards the latter.